construo/src/main/java/io/github/fourlastor/construo/ConstruoPluginExtension.kt

```diff
@@ -26,6 +27,14 @@ abstract class ConstruoPluginExtension @Inject constructor(
     val jlink: JlinkOptions = objectFactory.newInstance(JlinkOptions::class.java).apply {
         guessModulesFromJar.convention(true)
         includeDefaultCryptoModules.convention(true)
+
+        val version = JavaVersion.current()
+        multiReleaseVersion.convention(
+            if(version < JavaVersion.VERSION_1_9)
+                "base"
+            else
+                version.toString()
+        )
```

Comment:
the defaults need to be determined  in `CreateRuntimeImageTask`, as it's not necessarily true that running JVM == jlink JVM, something like:

```kotlin
    private fun getJavaVersionFromSdk(javaHome: File) = ByteArrayOutputStream().use {
        execOperations.exec {
            setWorkingDir(jdkRoot)
            standardOutput = it
            commandLine(
                File(javaHome, executableForOs("bin/javap")).absolutePath,
                "-version",
            )
        }
        it.toByteArray().toString(Charset.defaultCharset()).substringBefore('.')
    } 
  ```